spec/ameba/rule/documentation/documentation_admonition_spec.cr

```diff
@@ -53,7 +59,7 @@ module Ameba::Rule::Documentation
           past_date = (Time.utc - 21.days).to_s(format: "%F")
           expect_issue subject, <<-CRYSTAL, admonition: admonition
             # %{admonition}(#{past_date}): sth in the past
-            # ^{admonition} error: Found a #{admonition} admonition in a comment (21 days past)
+            # ^{admonition} error: Found a %{admonition} admonition in a comment (21 days past)
```

Comment:
could you please clarify what the difference in this syntax and we change it?

Reply:
There's none. I've changed it for consistency reasons, as I've already used the `%{}` variant.